src/math/src/sha512.cairo

```diff
@@ -8,6 +8,23 @@ pub const SHA512_LEN: usize = 64;
 
 pub const U64_BIT_NUM: u64 = 64;
 
+// Powers of two to avoid recomputing
+pub const TWO_56: u64 = 0x100000000000000;
```

Comment:
why not u128 and avoid .into() 
(even though it is a free op ^^')
Same for some other consts

Reply:
Because we also use it as u64, and this allows avoiding u128 `try_into()` u64

Reply:
![image](https://github.com/keep-starknet-strange/alexandria/assets/11048263/286fd178-0c41-4269-91d7-f09b2ef4c315)

Reply:
I had a couple `two_pow` calls which were using this before I cached all known powers of 2.
It'll be useful if we wanna use two_pow/shr (shl can be risky) somewhere with u64.

Reply:
Some are only used as u128, but yeah, as it is free op, nvm :)